deploy/kubernetes/clusterrole.yaml

```diff
@@ -74,6 +74,15 @@ rules:
   - list
   - patch
   - watch
+- apiGroups: ["authentication.istio.io"]
```

Comment:
cc @jmazzitelli @gbaufake this PR will add new changes in the clusterrole.yaml
I don't remember if we need to perform additional steps in order to propagate this once the PR is merged.

Reply:
We need to create a PR once these changes are in kiali and released as part of v0.14. I created a branch for this - when we are ready for upstream Istio to consume our v0.14 release, I'll create this PR:

https://github.com/istio/istio/compare/master...jmazzitelli:kiali-next-version-changes?expand=1